code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -95,6 +95,14 @@
 
 /obj/effect/proc_holder/spell/aoe_turf/rust_conversion/cast(list/targets, mob/user = usr)
 	playsound(user, 'sound/items/welder.ogg', 75, TRUE)
+	var/list/cuffies = list()
+	var/mob/living/carbon/heretic = usr
```

Comment:
the caster of this spell might not be a carbon

Reply:
it may or may not be interesting to have this break the cuffs of all nearby heretics and heretics monsters, in case rust walkers try to use this to break their master's cuffs or a heretic tries to use this to break the cuffs of one of their ghouls or voiceless dead.